gateway/modules/filestore/local/local.go

```diff
@@ -13,7 +14,10 @@ type Local struct {
 
 // Init initialises the local filestore driver
 func Init(path string) (*Local, error) {
-	return &Local{path}, nil
+	if !strings.HasPrefix(path, "/") {
```

Comment:
I think this is wrong. The rooth path can be relative too right?

Reply:
ok, then i'll remove this line